### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by joao-p-marques**
Fix bugs found in https://github.com/Tecnativa/ci-base/pull/3

### CODE REVIEW

File: src/.github/workflows/ci.yml.jinja [hunk 1]
```diff
@@ -116,7 +118,7 @@ jobs:
             context: .
             file: ./Dockerfile
             platforms: |
-            {%- for platform in platforms or ['linux/386','linux/amd64','linux/arm/v6','linux/arm/v7','linux/arm/v8','linux/arm64','linux/ppc64le','linux/s390x'] %}
+            {%- for platform in docker_platforms or ['linux/386','linux/amd64','linux/arm/v6','linux/arm/v7','linux/arm/v8','linux/arm64','linux/ppc64le','linux/s390x'] %}
```
Comment (yajo) [inline src/.github/workflows/ci.yml.jinja]: Just `platforms` or `image_platforms` is better.

Comment (joao-p-marques) [reply src/.github/workflows/ci.yml.jinja]: Thanks! Done

File: src/.github/workflows/ci.yml.jinja [hunk 2]
```diff
@@ -185,7 +187,5 @@ jobs:
             {%- raw %}
             labels: ${{ steps.docker_meta_public.outputs.labels }}
             tags: ${{ steps.docker_meta_public.outputs.tags }}
-            build-args: |
-              VCS_REF=${{ github.sha }}
```
Comment (joao-p-marques) [inline src/.github/workflows/ci.yml.jinja:189]: I was thinking that we should let `docker_meta` handle everything tag-related. Would removing this cause any problems?

Comment (joao-p-marques) [reply src/.github/workflows/ci.yml.jinja:189]: After investigating a bit more, it seems this works on a different level and we could leave them both. 
Is it ok for you @Yajo?

Comment (yajo) [reply src/.github/workflows/ci.yml.jinja:189]: > we should let `docker_meta` handle everything tag-related

Yes, that's how it must be done.

**Pull request merged**